ui/src/tugboat/components/form/Form.tsx

```diff
@@ -52,6 +65,28 @@ export const Form = <I, O>(props: Props) => {
         return fetch(url, opt).then((r) => r.json());
     };
 
+    // When the selected model changes we clear the current input. This doesn't reset the
+    // form values, but rather clears the output that's displayed so it's clear that it's
+    // no longer relevant.
+    React.useEffect(() => {
+        setInput(undefined);
+    }, [models.selectedModel]);
+
+    // Whenever the selected example changes, set the appropriate form values.
+    const examples = React.useContext(Examples);
+    React.useEffect(() => {
+        if (examples.selectedExample) {
+            // The wide cast here is unfortunate, but probably worth the tradeoff. The type `antd`
+            // expects here is a RecursivePartial<I>, but really `antd` should probably just expect
```

Comment:
```suggestion
            // The wide cast here is unfortunate, but probably worth the tradeoff. The type expected by
            // `antd` is a RecursivePartial<I>, but really `antd` should probably just expect
```

because I had to re-read "The type antd ..." to understand.